network/src/androidMain/kotlin/com/wire/kalium/network/OkhttpClientFactory.kt

```diff
@@ -35,8 +39,15 @@ private object OkHttpSingleton {
                 .writeTimeout(WEBSOCKET_TIMEOUT, TimeUnit.MILLISECONDS)
         }.connectionSpecs(supportedConnectionSpecs()).build()
     }
+    private val clearTextTrafficClient by lazy {
+        OkHttpClient.Builder().apply {
```

Comment:
please do not create multiple instances of okHttp you can reuser the same one

Reply:
something like 

```
fun buildClearTextTrafficOkhttpClient(): OkHttpClient = OkHttpSingleton.createNew {
        connectionSpecs(listOf(ConnectionSpec.CLEARTEXT))
    }
 ```
 
 and the `    private val clearTextTrafficClient by lazy {` can be deleted

Reply:
The current okHttp client has `RESTRICTED_TLS` spec, I created another one that have `CLEARTEXT` spec to allow HTTP connection